src/tests/asyncTests.ts

```diff
@@ -395,8 +361,8 @@ export class AsyncTests {
         queries.push(query);
       }
 
-      const queryPromises = queries.map((query) => {
-        return queryAll(this.connection, query, true);
+      const queryPromises = queries.map(async (query) => {
+        return queryAll(await this.defineApiVersion(), query, true);
```

Comment:
This will be called for each entry in queries list, capture the version outside of the .map as a const and then ref in the .map